### PR TITLE
adding bold  Labels for Yes/No Columns (e.g. Escalated/Priolist, Watchlist)

### PR DESCRIPTION
**Pull request by npalchur**
here is  the reference issue : 
https://github.com/RHsyseng/t5g-field-support-team-utils/issues/103

### CODE REVIEW

**Comment by dcritch** [comment]
Looks good, other than the issues Adhitya pointed out. Get those fixed up and we should be good to go. Thanks!

**Comment by npalchur** [comment]
thanks for the review , let me make those changes

File: dashboard/src/t5gweb/templates/macros/macros.html [hunk 1]
```diff
@@ -110,23 +110,23 @@ <h2>Statistics</h2>
                             {% if new_comments[account][status][card]['escalated'] %}
                                 {% if new_comments[account][status][card]['escalated_link'] %}
                                     <td class="align-middle text-center">
-                                        <a href="{{ new_comments[account][status][card]['escalated_link'] }}">Yes</a>
+                                        <a href="{{ new_comments[account][status][card]['escalated_link'] }} fw-bold">Yes</a>
```
Comment (adhil0) [inline dashboard/src/t5gweb/templates/macros/macros.html:113]: One last fix: fw-bold should go in the `<td>` class list on line 112

**Review by adhil0 — CHANGES_REQUESTED** [review]

**Review by adhil0 — APPROVED** [review]
Thanks!

**Pull request merged**